desktop/cmp/persistenceManager/impl/SaveDialogModel.ts

```diff
@@ -0,0 +1,86 @@
+import {FormModel} from '@xh/hoist/cmp/form';
+import {HoistModel, managed, PlainObject, TaskObserver, XH} from '@xh/hoist/core';
+import {lengthIs, required} from '@xh/hoist/data';
+import {makeObservable} from '@xh/hoist/mobx';
+import {PersistenceManagerModel} from '../PersistenceManagerModel';
+
+export class SaveDialogModel extends HoistModel {
+    parentModel: PersistenceManagerModel;
+
+    @managed saveTask = TaskObserver.trackLast();
+
+    @managed formModel = new FormModel({
```

Comment:
Would make `readonly`. Model config seems lengthy enough to break out into its own method (i.e. `@managed formModel = this.createFormModel()`